docs/data/data-grid/style/style.md

```diff
@@ -116,6 +116,25 @@ The following demo illustrates how this can be achieved.
 
 {{"demo": "StripedGrid.js", "bg": "inline"}}
 
+## Theme header & pinned sections
+
+The Data Grid will use by default the MUI `theme.palette.background.default` color for the background of its header and pinned sections, which require a solid color to hide the scrollable content behind them. You can override that color with the following configuration:
```

Comment:
```suggestion
The Data Grid will use by default the Material UI `theme.palette.background.default` color for the background of its header and pinned sections, which require a solid color to hide the scrollable content behind them. You can override that color with the following configuration:
```

Was just passing by and noticed his opportunity—Let's avoid using "MUI" as it is a confusing term (i.e., "MUI" is the company, "Material UI" is the library/product).

Reply:
I think it's fine to just mention Material UI. Wouldn't worry about Joy UI at the moment. Or possibly just mention MUI System, then, because either library uses it?

```suggestion
The Data Grid will use by default the MUI System `theme.palette.background.default` color for the background of its header and pinned sections, which require a solid color to hide the scrollable content behind them. You can override that color with the following configuration:
```

@samuelsycamore thoughts?